uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/leave-request-instance_test.js

```diff
@@ -410,5 +410,131 @@ define([
         });
       });
     });
+
+    describe('check status methods', function () {
+
+      var optionGroupDeferred,
+        mockOptionValue,
+        promise;
+
+      function commonSetup(statusName, methodName) {
```

Comment:
Please put this helper function at the end of the `describe`, let's use hoisting to clean up the code and make it more readable